playground/api/v1/api.proto

```diff
@@ -135,6 +135,16 @@ message GetLogsResponse {
   string output = 1;
 }
 
+// GetGraphRequest contains information of the pipeline uuid.
+message GetGraphRequest {
+  string pipeline_uuid = 1;
+}
+
+// GetGraphResponse represents the graph of the executed code.
+message GetGraphResponse {
+  string graph = 1;
```

Comment:
can you document in what format the graph is returned? Is this a file address? Is it a byte string with the graph image formatted in some way? is it a markdown-type graph format?

Reply:
I changed this comment. Overall `graph` is a string representation like this one: https://github.com/mehmandarov/word-count-mini-beam/blob/master/pipeline_graph.dot
Using this string frontend can build and show the graph on the page.

Reply:
can you mention it's in DOT format? so that it's clear for anyone reading this (or at least they can google it)

Reply:
Done.